### PR TITLE
229 fix logging for metadata

### PR DESCRIPTION
**Pull request by ElBabbler**
Adding ComponentLogger to verifyCredentials and metadata uploading context

### CODE REVIEW

File: lib/service.js [hunk 1]
```diff
@@ -147,7 +148,9 @@ function processService(serviceMethod, env) {
                     }
                     resolve(result);
                 }
-                const result = verify(cfg, legacyCallback);
+                var callScope = new EventEmitter();
+                callScope.logger = new ComponentLogger({});
```
Comment (andkom) [inline lib/service.js]: are you sure we need to pass empty object?

Comment (ElBabbler) [reply lib/service.js]: To awoid NPE?

Comment (andkom) [reply lib/service.js]:
```
Logger.prototype.child = function (options, simple) {
    return new (this.constructor)(this, options || {}, simple);
}
```
you can skip options argument

Comment (ElBabbler) [reply lib/service.js]: Ok, changed it

**Pull request merged**